users.tf

```diff
@@ -0,0 +1,15 @@
+module "users" {
+  source  = "dasmeta/modules/aws//modules/aws-iam-user"
+  version = "0.29.1"
+
+  for_each = { for user in var.users : user.username => user }
+
+  username          = each.value.username
+  policy_attachment = each.value.policy_attachment
+}
+
+module "user-group" {
+  source = "dasmeta/modules/aws//modules/iam-group"
+  name   = "test-account-module"
```

Comment:
have module versions always set,
do not hardcode module chancheble variables,
do not use "test" word as default values for modules,

do we need this module?